### PR TITLE
Update links for v2 API.  The existing v1 documents will be hosted

### PR DESCRIPTION
**Pull request by kevinrbabcock**
on v1.developer.pagerduty.com, and the new v2 documents will be hosted
on developer.pagerduty.com.

### CODE REVIEW

File: source/index.html [hunk 1]
```diff
@@ -58,7 +58,7 @@
 </div>
 <div class='section'>
                       <h3>Want to be part of BetaDuty?</h3>
-                     Fill out this <a href="https://developer.pagerduty.com/beta">form</a> and we will contact you when a Beta of your choice opens! 
+                     Fill out this <a href="https://v1.developer.pagerduty.com/beta">form</a> and we will contact you when a Beta of your choice opens!
```
Comment (billkauf) [inline source/index.html]: @steverice @eurica  Will we have a new equivalent to the "BetaDuty" program?

File: _config.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Site settings
-title: PagerDuty Developer Documentation
+title: PagerDuty Developer Documentation v1
 email: developers@pagerduty.com
-description: The official site for PagerDuty API documentation.
-url: "https://developer.pagerduty.com"
+description: The official site for PagerDuty APIv1 documentation.
```
Comment (billkauf) [inline _config.yml]: Maybe put a space in there--`API v1`?

**Comment by billkauf** [comment]
Might be worth adding a "V1" [in the nav column header](https://github.com/PagerDuty/devdocs/blob/master/source/_includes/header.html#L49) as well.

**Comment by billkauf** [comment]
LGTM

File: _config.yml (hunk 2 above)
Comment (kevinrbabcock) [inline _config.yml]: Done

**Comment by kevinrbabcock** [comment]
I added v1 to the column header and fixed the release notes on the main page.

**Pull request merged**